jwst/cube_build/src/cube_match_sky_driz.c

```diff
@@ -283,9 +283,9 @@ int match_driz(double *xc, double *yc, double *zc,
 		  weightv[index_cube] = weightv[index_cube] + area_weight;
 		  varv[index_cube] = varv[index_cube] + weighted_var;
 		  ifluxv[index_cube] = ifluxv[index_cube] +1.0;
-		  if(index_cube == 1577){
-		    printf(" found value %f %f %f %f \n", area_weight, zoverlap, weighted_flux, flux[k]);
-		  }
+		  //if(index_cube == 1577){
+		  //  printf(" found value %f %f %f %f \n", area_weight, zoverlap, weighted_flux, flux[k]);
+		  //}
```

Comment:
Yes - it was printing out information to the screen so I  commented it out

Reply:
If done with debugging statements, it is preferable that they be removed completely.  If there is felt to leave the code, add commentary as to why/what purpose the code is "remembered" for.

Reply:
Agree that these should be completely removed, because they're so specific to one particular use case. If they were more generic they could be left in as debug-level log messages.